src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE = 0;
+    private static final int AMOUNT = 1;
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method `getStatistic` is responsible for both reading from a file and writing to a file. According to the single responsibility principle, a method should do one thing and do it well. It's better to divide this logic into separate methods, for instance, one for reading the file and calculating statistics, and another one for writing the statistics to the file. This is a code quality issue.